File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/timer/TimerActivity.java
package com.jdroid.android.sample.ui.timer;

import androidx.fragment.app.Fragment;

import com.jdroid.android.activity.FragmentContainerActivity;

public class TimerActivity extends FragmentContainerActivity {

	@Override
	protected Class<? extends Fragment> getFragmentClass() {
		return TimerFragment.class;
	}
}

File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/timer/TimerActivity.kt
package com.jdroid.android.sample.ui.timer

import androidx.fragment.app.Fragment

import com.jdroid.android.activity.FragmentContainerActivity

class TimerActivity : FragmentContainerActivity() {

    override fun getFragmentClass(): Class<out Fragment> {
        return TimerFragment::class.java
    }
}
